orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -234,6 +252,7 @@ private static boolean checkValidSpeciesProtein(String homologueId) {
 				referenceDNAInstances.add(alternateRefDNAInst);
 			}
 		}
+		logger.info("Total ReferenceDNASequence instance(s) created: " + referenceDNAInstances.size());
```

Comment:
I like the idea of info messages giving information on what's been created, and how many. :+1: